site/content/en/docs/Installation/upgrading.md

```diff
@@ -51,6 +51,70 @@ The following are steps to implement this:
 If you are upgrading a single cluster, we recommend creating a maintenance window, in which your game goes offline
 for the period of your upgrade, as there will be a short period in which Agones will be non-responsive during the upgrade.
 
+{{% feature publishVersion="1.99.0" %}}
+#### In-Place Agones Upgrades
```

Comment:
Do we want to publish this now, or wait for the release? There's no feature gate, so there's nothing preventing a user from starting to use in place upgrades now.

Reply:
I think it's still make sense to have a defined time point when the feature is "officially" in alpha. So let's keep this.